### PR TITLE
Update readme with steps to execute integration tests

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Update readme to include steps to execute integration tests.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/243 |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Ran the integration tests using the steps mentioned in the readme.
Screenshots:
Module whose testcases ran successfully:
image
Module whose testcase failed:
image
Sample of Coverage data rendered to HTML after execution of step 5 (in Integration test execution section):
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # gounity
 A portable Go library for unity related operations such as create-volume, list-volume, etc.__
```
Comment (rensyct) [inline README.md:2]: Please add relevant screenshots which would help the user. Please ensure to modify the Array details and internal IPs

Comment (panigs7) [reply README.md:2]: Done

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+
+## Integration Tests execution
+Follow the steps to run integration tests:
+1. Create a properties file `test.properties` using the template `test.properties_template`.
+2. Populate the properties file with values for your DellEMC Unity system as shown below:
```
Comment (rensyct) [inline README.md:7]: Remove DellEMC
Unity XT system

Comment (panigs7) [reply README.md]: Done

**Review by karthikk92 — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Pull request merged**